content/docs/reference/pkg/nodejs/pulumi/keycloak/Ldap/_index.md

```diff
@@ -0,0 +1,2112 @@
+---
+
+<!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
+<!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/tscdocgen. -->
+
+
+> This provider is a derived work of the [Terraform Provider](https://github.com/mrparkers/terraform-provider-keycloak)
+> distributed under [MIT](https://mit-license.org/). If you encounter a bug or missing feature,
+> first check the [`pulumi/pulumi-keycloak` repo](https://github.com/pulumi/pulumi-keycloak/issues); however, if that doesn't turn up anything,
+> please consult the source [`mrparkers/terraform-provider-keycloak` repo](https://github.com/mrparkers/terraform-provider-keycloak/issues).
+
+
+
+
+
+<h3>Resources</h3>
+<ul class="api">
+    <li><a href="#FullNameMapper"><span class="symbol resource"></span>FullNameMapper</a></li>
+    <li><a href="#GroupMapper"><span class="symbol resource"></span>GroupMapper</a></li>
+    <li><a href="#MsadUserAccountControlMapper"><span class="symbol resource"></span>MsadUserAccountControlMapper</a></li>
+    <li><a href="#UserAttributeMapper"><span class="symbol resource"></span>UserAttributeMapper</a></li>
+    <li><a href="#UserFederation"><span class="symbol resource"></span>UserFederation</a></li>
+</ul>
+
+
+<h3>Others</h3>
+<ul class="api">
+    <li><a href="#FullNameMapperArgs"><span class="symbol api"></span>FullNameMapperArgs</a></li>
+    <li><a href="#FullNameMapperState"><span class="symbol api"></span>FullNameMapperState</a></li>
+    <li><a href="#GroupMapperArgs"><span class="symbol api"></span>GroupMapperArgs</a></li>
+    <li><a href="#GroupMapperState"><span class="symbol api"></span>GroupMapperState</a></li>
+    <li><a href="#MsadUserAccountControlMapperArgs"><span class="symbol api"></span>MsadUserAccountControlMapperArgs</a></li>
+    <li><a href="#MsadUserAccountControlMapperState"><span class="symbol api"></span>MsadUserAccountControlMapperState</a></li>
+    <li><a href="#UserAttributeMapperArgs"><span class="symbol api"></span>UserAttributeMapperArgs</a></li>
+    <li><a href="#UserAttributeMapperState"><span class="symbol api"></span>UserAttributeMapperState</a></li>
+    <li><a href="#UserFederationArgs"><span class="symbol api"></span>UserFederationArgs</a></li>
+    <li><a href="#UserFederationState"><span class="symbol api"></span>UserFederationState</a></li>
+</ul>
+
+
+<h2 id="resources">Resources</h2>
+<h3 class="pdoc-module-header" id="FullNameMapper" data-link-title="FullNameMapper">
+    <a href="https://github.com/pulumi/pulumi-keycloak/blob/{{< param git_sha >}}/sdk/nodejs/Ldap/fullNameMapper.ts#L60">
+        Resource <strong>FullNameMapper</strong>
+    </a>
+</h3>
+
+<pre class="highlight"><code><span class='kr'>class</span> <span class='nx'>FullNameMapper</span> <span class='kr'>extends</span> <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#CustomResource'>CustomResource</a></code></pre>
+
+#### # keycloak.Ldap.FullNameMapper
```

Comment:
This is probably going to look odd. Seems like another case where we could improve the doc extraction.

Reply:
👍 I don't think many eyes will be on this set of docs for a while - so we have time :)